bld/namelist_files/namelist_defaults_cmip_era.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<namelist_defaults>
+
+<!--
+Values to use by default for creation of CLM model driver namelists.
+The element names are the same as the corresponding namelist
+variables.  Values that depend on the model configuration use
+attributes to express the dependency.  The recognized attributes
+are: RESOLUTION, defaults, mask, ic_ymd, ic_tod, sim_year and all configuration
+attributes from the config_cache.xml file (with keys converted to upper-case).
+-->
+
+<!-- Defaults for cmip_era_inparm namelist  -->
+<cmip_era>cmip7</cmip_era>
+<cmip_era sim_year="PtVg" >cmip6</cmip_era>
+<cmip_era sim_year_range="1850-2100" >cmip6</cmip_era>
+<cmip_era sim_year_range="2000-2100" >cmip6</cmip_era>
+<cmip_era sim_year_range="2000-2025" >cmip6</cmip_era>
```

Comment:
It looks to me like 2000-2025 could be removed. And looking at other sim_year_range values it looks like many of them can be removed

So remove the line regarding it. And I'll open an issue about removing more sim_year_range options.